src/darwin/Framework/CHIP/MTRDevice+XPC.mm

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
MTRDevice_XPCHelpers/Additions/Tools/Utilities/Protocols?

I assume this will be different than the MTRDevice_XPC.m object?

Reply:
This is something I just started realizing needs to be different - there's the proxy service side and there's the proxy object that the client will use rather than just using the proxy service object directly, i.e. to get a synchronous API to match existing MTRDevice API.  Thus, the time for raising MTRDevice / lowering its implementation has come.  Will discuss with you shortly.